snippets/csharp/VS_Snippets_CLR_System/system.Security.Permissions.UrlIdentityPermission/CS/urlidentity.cs

```diff
@@ -51,29 +51,17 @@ private static void UnionDemo()
         UrlIdentityPermission permIdPerm1 = new UrlIdentityPermission("http://www.fourthcoffee.com/process/");
         UrlIdentityPermission permIdPerm2 = new UrlIdentityPermission("http://www.fourthcoffee.com/*");
         UrlIdentityPermission p3 = (UrlIdentityPermission)permIdPerm1.Union(permIdPerm2);
```

Comment:
This is throwing a NotSupportedException. Despite the documentation, this seems to only succeed if given two equal URLs. At the same time, the functionality is not supported in .NET Framework 4 and later. But the example is not used. The UnionDemo method can be removed from the source code (and the call to UnionDemo on line 12 should also be removed).